docs/src/engines/lammps.rst

```diff
@@ -227,6 +228,8 @@ configure the build with:
     # optionally install the code on your machine. You can also directly use
     # the `lmp` binary in `lammps-metatomic/build/lmp` without installation
     cmake --build . --target install # or `make install`
+    # For usage with lammps within ase
+    make install-python
```

Comment:
With the configuration marked above, this will not work since we are not enabling the Python side of things. So I would rather not put anything here and refer people to the official documentation for anything not related to metatomic

Reply:
Yeap makes sense.